.github/workflows/rebase.yml

```diff
@@ -17,6 +17,15 @@ jobs:
     if: github.event.issue.pull_request != '' && github.event.comment.body == 'GitHub, rebase'
     runs-on: ubuntu-latest
     steps:
+      - uses: khan/pull-request-comment-trigger@master
+        name: React with rocket on run
+        with:
+          trigger: 'GitHub, rebase'
```

Comment:
Why it is here? This step will be executed only if comment is "Github, rebase", so no need for this

Reply:
I think it is `trigger : 'GitHub, rebase` actually sends the comment text to match with "GitHub, rebase" so if it is not there script doesn't runs.

Reply:
have you tried it without this line?

Reply:
Yes. I also tested it with just a single space instead of total 'GitHub, rebase' but that too doesn't work.

Reply:
will it work without trigger at all? because there is no need in trigger, let it work everytime

Reply:
Triggered is a required parameter [Inputs](https://github.com/Khan/pull-request-comment-trigger#inputs) .